include/aspect/gravity_model/radial.h

```diff
@@ -135,6 +135,52 @@ namespace aspect
         double magnitude_at_bottom;
 
     };
+
+
+
+    template <int dim>
+    class RadialLinearWithTidalPotential : public RadialLinear<dim>
+    {
+      public:
+        /**
+         * Return the gravity vector as a function of position.
+         */
+        Tensor<1,dim> gravity_vector (const Point<dim> &position) const override;
+
+        /**
+         * Declare the parameters this class takes through input files.
+         */
+        static
+        void
+        declare_parameters (ParameterHandler &prm);
+
+        /**
+         * Read the parameters this class declares from the parameter file.
+         */
+        void
+        parse_parameters (ParameterHandler &prm) override;
+
+      private:
+        /**
+         * Mass of hosting planet
```

Comment:
what is a "hosting" planet? Do you mean the mass of the planet/star that generates the tidal potential? That would be worth spelling out.